commit 44d8c4d516d951ac29a5f2cab7788002d6a77211
Author: Dmitry Baryshkov <lumag@kernel.org>
Date:   Tue Oct 10 08:49:20 2023 +0300

    drm/msm: fix fault injection support
    
    Readd the header that was erroneously dropped during KMS code
    refactoring.
    
    Fixes: 506efcba3129 ("drm/msm: carve out KMS code from msm_drv.c")
    Reported-by: kernel test robot <lkp@intel.com>
    Closes: https://lore.kernel.org/oe-kbuild-all/202310100836.6E6zjEcE-lkp@intel.com/
    Signed-off-by: Dmitry Baryshkov <dmitry.baryshkov@linaro.org>
    Patchwork: https://patchwork.freedesktop.org/patch/561739/
    Signed-off-by: Rob Clark <robdclark@chromium.org>

diff --git a/drivers/gpu/drm/msm/msm_drv.c b/drivers/gpu/drm/msm/msm_drv.c
index 401e9ef86074..b61ccea05327 100644
--- a/drivers/gpu/drm/msm/msm_drv.c
+++ b/drivers/gpu/drm/msm/msm_drv.c
@@ -6,6 +6,7 @@
  */
 
 #include <linux/dma-mapping.h>
+#include <linux/fault-inject.h>
 #include <linux/of_address.h>
 #include <linux/uaccess.h>